mycpp/tuple_types.h

```diff
@@ -34,6 +48,7 @@ class Tuple3 {
   }
 
  private:
+  OBJ_HEADER();
```

Comment:
I think we should be more consistent about inheriting from Obj vs OBJ_HEADER(), which are roughly equivalent ways to do the same thing

The problem is multiple inheritance, which we use in ASDL

But here there is no inheritance so I think it's OK to inherit from Obj ?

Reply:
Again: If you can figure out how to get the compiler to accept Tuple inheriting from Obj, be my guest. I tried to get it to cooperate and it would not